sktime/transformations/series/dilation_mapping.py

```diff
@@ -0,0 +1,117 @@
+"""DilationMapping transformer.
+
+A transformer for applying dilation mapping to time series data.
+"""
+
+__author__ = ["fspinna"]
+__all__ = ["DilationMappingTransformer"]
+
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+
+
+class DilationMappingTransformer(BaseTransformer):
+    r"""DilationMapping transformer.
+
+    A transformer for applying dilation mapping to time series data, as seen in:
```

Comment:
not blocking, but it would be very useful for the user to explain what it does, mathematically - after all, it is not very complicated. Perhaps just one line or two of math? "dilation" is not a standing term.